docs/app/pages/components/sections/utilities/focus-if/focus-if.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="focus-demo-box aspects-focus-outline" tabindex="0" [focusIf]="focused">
+    <p class="p-sm">Click the button below to focus on this box</p>
+</div>
+
+<button class="btn button button-primary m-t-md" (click)="focused = true">Click</button>
+
+<hr>
+
+<p>The focus if directive allows you to set focus on an element when the conditions are met, 
```

Comment:
Should possibly change to:

```html
<p>The <code>focusIf</code> directive allows you to...
```

It might also be good to mention somewhere that it should be used as an attribute as a directive can technically be an element or several other things.

Possibly also reword: "when the conditions are met" to pretty much say when the attribute value is true the element will gain focus, as it does not necessarily have to be used with a condition.